homeassistant/components/modbus/__init__.py

```diff
@@ -463,6 +463,9 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
 async def async_reset_platform(hass: HomeAssistant, integration_name: str) -> None:
     """Release modbus resources."""
+    if DOMAIN not in hass.data:
+        _LOGGER.info("Modbus cannot reload, because it was never loaded")
```

Comment:
Isn't this worth an error?

Reply:
ups

Reply:
Copy/paste error (mine), corrected, thanks for catching.